contrib/to-a11y-string/to-a11y-string.js

```diff
@@ -0,0 +1,735 @@
+    "\\tan": "tangent",
+    "\\cot": "cotangent",
+    "\\sum": "sum",
+    "/": "slash",
+    ",": "comma",
+    ".": "point",
+    "-": "negative",
+    "+": "plus",
+    "~": "tilde",
+    ":": "colon",
+    "?": "question mark",
+    "'": "apostrophe",
+    "\\%": "percent",
+    " ": "space",
+    "\\ ": "space",
+    "\\$": "dollar sign",
+    "\\angle": "angle",
+    "\\degree": "degree",
+    "\\circ": "circle",
+    "\\vec": "vector",
+    "\\triangle": "triangle",
+    "\\pi": "pi",
+    "\\prime": "prime",
+    "\\infty": "infinity",
+    "\\alpha": "alpha",
+    "\\beta": "beta",
+    "\\gamma": "gamma",
+    "\\omega": "omega",
+    "\\theta": "theta",
+    "\\sigma": "sigma",
+    "\\lambda": "lambda",
+    "\\tau": "tau",
+    "\\Delta": "delta",
+    "\\delta": "delta",
+    "\\mu": "mu",
+    "\\rho": "rho",
+    "\\nabla": "del",
+    "\\ell": "ell",
+    "\\ldots": "dots",
+    // TODO: add entries for all accents
+    "\\hat": "hat",
+};
+
+const powerMap = {
+    "prime": "prime",
+    "degree": "degrees",
+    "circle": "degrees",
+    "2": "squared",
+    "3": "cubed",
+};
+
+const openMap = {
+    "|": "open vertical bar",
+    ".": "",
+};
+
+const closeMap = {
+    "|": "close vertical bar",
+    ".": "",
+};
+
+const binMap = {
+    "+": "plus",
+    "-": "minus",
+    "\\pm": "plus minus",
+    "\\cdot": "dot",
+    "*": "times",
+    "/": "divided by",
+    "\\times": "times",
+    "\\div": "divided by",
+    "\\circ": "circle",
+    "\\bullet": "bullet",
+};
+
+const relMap = {
+    "=": "equals",
+    "\\approx": "approximately equals",
+    "≠": "does not equal",
+    "\\geq": "is greater than or equal to",
+    "\\ge": "is greater than or equal to",
+    "\\leq": "is less than or equal to",
+    "\\le": "is less than or equal to",
+    ">": "is greater than",
+    "<": "is less than",
+    "\\leftarrow": "left arrow",
+    "\\Leftarrow": "left arrow",
+    "\\rightarrow": "right arrow",
+    "\\Rightarrow": "right arrow",
+    ":": "colon",
+};
+
+const accentUnderMap = {
+    "\\underleftarrow": "left arrow",
+    "\\underrightarrow": "right arrow",
+    "\\underleftrightarrow": "left-right arrow",
+    "\\undergroup": "group",
+    "\\underlinesegment": "line segment",
+    "\\utilde": "tilde",
+};
+
+type NestedArray<T> = Array<T | NestedArray<T>>;
+
+const buildString = (
+    str: string,
+    type: Atom | "normal",
+    a11yStrings: NestedArray<string>,
+) => {
+    if (!str) {
+        return;
+    }
+
+    let ret;
+
+    if (type === "open") {
+        ret = str in openMap ? openMap[str] : stringMap[str] || str;
+    } else if (type === "close") {
+        ret = str in closeMap ? closeMap[str] : stringMap[str] || str;
+    } else if (type === "bin") {
+        ret = binMap[str] || str;
+    } else if (type === "rel") {
+        ret = relMap[str] || str;
+    } else {
+        ret = stringMap[str] || str;
+    }
+
+
+    // If nothing was found and it's not a plain string or number
```

Comment:
We longer need to test this?

Reply:
Let me double check this.

Reply:
This check is too restrictive.  KaTeX accepts pretty much all Unicode now.  I'll remove this commented out code.